lib/internal/abort_controller.js

```diff
@@ -357,6 +362,24 @@ function transferableAbortController() {
   return AbortController[kMakeTransferable]();
 }
 
+/**
+ * @param {AbortSignal} signal
+ * @param {any} resource
+ * @returns {Promise<void>}
+ */
+function aborted(signal, resource = null) {
```

Comment:
Any reason this isn't an async function? That would let you just do `abortSignalValidator` instead of doing the try/catch dance here.

Reply:
No particular reason, can replace with a async one

Reply:
Or throwing synchronously would maybe makes sense. Node.js API is a bit inconsistent on this, but +1 for not doing the try/catch here, wether in an async or a sync function.

Reply:
Made it async and removed the try-catch usage